sandy/decay.py

```diff
@@ -132,6 +132,66 @@ def get_halflives(self):
         series.index.name = "ZAM"
         return series
 
+    def get_br(self, with_uncertainty=True):
+        """
+        Extract branching ratio and its uncertainty into a dataframe.
+
+        Parameters
+        ----------
+        with_uncertainty : `bool`, optional, default is 'True'
+            makes the method return branching ratios and uncertainties
+            if set equal True, or else return only the branching ratios
+
+        Returns
+        -------
+        `pandas.DataFrame`
+            dataframe with branching ratio and associated uncertainty or
+            dataframe with branching ratio
+
+        Examples
+        --------
+        >>> endf6 = sandy.get_endf6_file("jeff_33", "decay", [942400, 922350])
+        >>> rdd = sandy.DecayData.from_endf6(endf6)
+        >>> BR = rdd.get_br()
+        >>> print(BR)
+                                    BR         DBR
+        ZAM    RTYP                               
+        922350 4.00000e+00 1.00000e+00 1.00000e-04
+               6.00000e+00 7.20000e-11 2.10000e-11
+        942400 4.00000e+00 1.00000e+00 0.00000e+00
+               6.00000e+00 5.70000e-08 2.00000e-09
+
+        >>> endf6 = sandy.get_endf6_file("jeff_33", "decay", [942400, 922350])
+        >>> rdd = sandy.DecayData.from_endf6(endf6)
+        >>> BR = rdd.get_br(False)
+        >>> print(BR)
+                                    BR
+        ZAM    RTYP                   
+        922350 4.00000e+00 1.00000e+00
+               6.00000e+00 7.20000e-11
+        942400 4.00000e+00 1.00000e+00
+               6.00000e+00 5.70000e-08
+        """
+        br = []
+        ZAM = []
+        RTYP = []
+        for zam, dic in self.data.items():
+            for rtyp, dk in dic['decay_modes']:
+                br.append([
+                    dk['branching_ratio'],
+                    dk['branching_ratio_uncertainty'],
+                    ])
+                RTYP.append(float(rtyp) / 10)
+                ZAM.append(zam)
```

Comment:
Why is rtyp a float? Can it be a int? Why do you divide by 10?

Reply:
It is usually a float for multiple particle decay and I divided by 10 to get the rtyp value stored in ENDF6. To solve it I changed the method read_rdd

Reply:
read_rdd and mf8 must not be changed this way.
Please, restore them as they originally were.

Check again what rtyp is on the endf6 manual.
As we have already discussed, if you divide by 10 you lose info on the gamma decay...it also won't work for delayed neutron precursors